lib/install/management/delete.go

```diff
@@ -288,6 +314,7 @@ func (d *Dispatcher) DeleteVCHInstances(vmm *vm.VirtualMachine, conf *config.Vir
 				errs = append(errs, err.Error())
 				mu.Unlock()
 			}
+			d.op.Debugf("Successfully deleted container vm: %s", child.InventoryPath)
```

Comment:
Minor: s/container vm/container (since we've referred to a `containerVM` as a `container` in the above log messages.